CHANGELOG.md

```diff
@@ -13,6 +13,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Deprecated`
 
+## [2.4.6] - 2022-11-14
```

Comment:
What's going on here?

Reply:
The CHANGELOG for dsl2 was branched off before 2.4.6 became a thing.